cognite/client/data_classes/simulators.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any
+
+from typing_extensions import Self
+
+from cognite.client.data_classes._base import (
+    CogniteObject,
+)
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+
+@dataclass
+class SimulationValueUnitName(CogniteObject):
+    name: str
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
+        return cls(
+            name=resource["name"],
+        )
+
+    def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        return super().dump(camel_case=camel_case)
+
+
+@dataclass
+class SimulationInputOverride(CogniteObject):
```

Comment:
What is the motivation to move this to a separate .py file?

Is it expected to be used with the simulator endpoints?

Reply:
exactly. there will be many more simint classes added, @lpereiracgn is working on another PR for that

Reply:
Does it make sense to move this into `/data_classes/simulators/<something>.py`? See for example what we did for `hosted_extractors/`. The thinking is that with more resources we start to get overlapping names, so we want to separate them into "namespace" which in python would be different packages.

Reply:
moved